### DISCUSSION TITLE
so-pcap-export error: Failed to connect to 127.0.0.1 port 1234

### DISCUSSION THREAD

**Discussion by UMHB-InfoSec** [Security-Onion-Solutions/securityonion#5342] [Unsupported Versions] [answered]
While looking for a way to export some pcaps I found the so-pcap-export tool that looks like it was added in 2.3.70.

1) it looks like the perfect tool so BIG thank you to the SO team!

but ... I can't get it to work.

```
[support@SOMaster ~]$ sudo so-pcap-export "host 10.10.10.10 after 1h ago" test
curl: (7) Failed to connect to 127.0.0.1 port 1234 after 0 ms: Connection refused
tcpdump: truncated dump file; tried to read 4 file header bytes, only got 0

If successful, the output was written to: /nsm/pcapout/test.pcap
```

The port 1234 looked funny but I decided to check and looking in `sudo netstat -nla | grep 1234` doesn't show anything listening on that port.  Is there some config file I am missing?

Also ... I could not find any documentation on this command.  Am I correct in my assumption that the tool will extract the relevant packets from the existing pcaps on my distributed forward nodes?

**Comment by dougburks** [comment, marked as answer]
so-pcap-export is just a very thin wrapper around stenoquery:
https://github.com/Security-Onion-Solutions/securityonion/blob/master/salt/common/tools/sbin/so-pcap-export

Because of this, it must be run directly on the node where the pcap is stored (not the manager).

I've updated the Stenographer page in our documentation:
https://docs.securityonion.net/en/2.3/stenographer.html#command-line
